addons/point_of_sale/static/src/app/models/pos_order_line.js

```diff
@@ -670,7 +670,11 @@ export class PosOrderline extends Base {
             ),
             taxGroupLabels: [
                 ...new Set(
-                    this.product_id.taxes_id
+                    getTaxesAfterFiscalPosition(
+                        this.product_id.taxes_id,
+                        this.order_id.fiscal_position_id,
+                        this.order_id.models
```

Comment:
```suggestion
                        this.models
```

Models is accessible from every model instances